x-pack/test/functional/services/aiops/explain_log_rate_spikes.ts

```diff
@@ -56,9 +58,93 @@ export function ExplainLogRateSpikesProvider({ getService }: FtrProviderContext)
       await testSubjects.existOrFail(`aiopsNoWindowParametersEmptyPrompt`);
     },
 
+    async assertNoResultsFoundEmptyPromptExist() {
+      await testSubjects.existOrFail(`aiopsNoResultsFoundEmptyPrompt`);
+    },
+
+    async clickDocumentCountChart() {
+      await elasticChart.waitForRenderComplete();
+      const el = await elasticChart.getCanvas();
+
+      await browser.getActions().move({ x: 0, y: 0, origin: el._webElement }).click().perform();
+    },
```

Comment:
Ideally, we would make sure that the click actually did something. Not sure what could be checked here though - does the click change something that we could assert on?

Reply:
After the click the brush interface should show up, I added assertions to make sure it's present in [4087ba8](https://github.com/elastic/kibana/pull/138661/commits/4087ba85d52c01e765cd07458bdf105131b23647).